### PR TITLE
Create aerial-tick-colors

### PR DESCRIPTION
**Pull request by JadeGerbitz**
My plugin allows the user to select different colors to highlight the fishing spots at aerial fishing based on the number of ticks it will take to catch the fish. The fishing plugin does this already, but it only has colors for 2-tick spots and not 2-tick spots. I think this should help visualize the distance to aerial fishing spots better.

### CODE REVIEW

**Comment by skeldoor** [comment]
Hi, did you forget to commit/push your changes? This looks like a blank template example plugin. Only the readme and the license is changed from the original.

These steps are worth following to get everything set up. https://github.com/runelite/plugin-hub#creating-new-plugins

Also is the repo name "Aeirial-Tick-Colors" a misspelling of "Aerial-Tick-Colors"

**Comment by JadeGerbitz** [comment]
The plugin code has been pushed to the repository and the typo in the repository name has been fixed.

**Comment by Nightfirecat** [comment]
You need to update the commit in this PR as well, as that's what would be built for the hub.

**Comment by JadeGerbitz** [comment]
I've updated the commit.

**Comment by Nightfirecat** [comment]
There already exists a hub plugin with the name "Aerial Fishing": https://runelite.net/plugin-hub/show/aerial-fishing. Your config group doesn't conflict with it, but we'd prefer you change it to be more specific to the full plugin name so as not to be confused, eg. `@ConfigGroup("aerial-tick-colors")`

**Comment by JadeGerbitz** [comment]
I've changed the config group to "Aerial-Tick-Colors".

File: plugins/aerial-tick-colors [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/JadeGerbitz/Aeirial-Tick-Colors.git
```
Comment (Felanbird) [inline plugins/aerial-tick-colors:1]: This is just a nit-pick but the repo url probably should be the post name-changed version
```suggestion
repository=https://github.com/JadeGerbitz/Aerial-Tick-Colors.git
```

**Pull request merged**